lib/base/exception.cpp

```diff
@@ -424,6 +431,39 @@ std::string icinga::to_string(const StackTraceErrorInfo&)
 }
 
 #ifdef _WIN32
+const char *win32_error::what() const noexcept
+{
+	return "win32_error";
+}
+
+std::string icinga::to_string(const win32_error &e) {
```

Comment:
Given that you've carried this over from my PR, does this mean that you're happy with this as-is?

Reply:
I consider it fine.